src/account/Persona.tsx

```diff
@@ -47,24 +47,28 @@ const Persona = ({ kycStatus, text, onCancelled, onError, onPress, onSuccess }:
     }
 
     if (!walletAddress) {
-      // accountMTWAddress can be null if user's phone number is not verified.
-      // Current plan for initial PFP release is to monitor drop off rate for users who haven't verified their phone numbers yet
-      // Discussion -> https://valora-app.slack.com/archives/C025V1D6F3J/p1637606953112000
-      Logger.warn(TAG, "Can't render Persona because accountMTWAddress is null")
+      Logger.warn(TAG, "Can't render Persona because walletAddress is null")
       return
     }
     onPress?.()
     Inquiry.fromTemplate(templateId)
       .referenceId(walletAddress)
       .environment(networkConfig.personaEnvironment)
       .iosTheme(pjson.persona.iosTheme)
-      .onSuccess((inquiryId: string, attributes: InquiryAttributes) => {
-        onSuccess?.(attributes?.address)
-        ValoraAnalytics.track(CICOEvents.persona_kyc_success)
-        Logger.info(TAG, `Inquiry completed for ${inquiryId}`)
+      .onComplete((inquiryId: string, status: string, _fields: Fields) => {
+        // status can be completed or failed
+        if (status === 'failed') {
```

Comment:
wish there was an exported enum for this :frowning_face:

Reply:
yeah I couldn't find one, but I'll look again one more time.